src/json-ld.tsx

```diff
@@ -39,13 +39,14 @@ import { Thing, WithContext } from "schema-dts";
  */
 export class JsonLd<T extends Thing> extends React.Component<{
   item: WithContext<T>;
+  space?: string | number;
```

Comment:
Can you as a jsdoc above this line?

Reply:
Just pushed a commit, is that what you had in mind or?

Reply:
My bad for not elaborating: you can put inline JSDocs above a type declaration (i.e. right above "space:") describing it, and it shows up in IDEs etc when someone's looking at props or seeing the autocomplete for space

Reply:
![image](https://user-images.githubusercontent.com/6671343/78580413-a75fe900-7832-11ea-9494-91c164571b8e.png)

`string | number` is used as type declaration here instead of JSDoc, no? I don't think we can write JSDoc in inline interface declaration like here, IDE is complaining and TS won't compile. Can you check?

Reply:
Here's what I mean:

![image](https://user-images.githubusercontent.com/554424/78591656-c8641200-7811-11ea-9c41-e836399d8c35.png)

![image](https://user-images.githubusercontent.com/554424/78591806-02cdaf00-7812-11ea-9195-c04e437f0434.png)